### PR TITLE
separate manual trigger project

### PR DESCRIPTION
**Pull request by meeyun-deriv**

### CODE REVIEW

File: .github/workflows/cypress_cloud_manual.yml [hunk 1]
```diff
@@ -44,13 +44,15 @@ jobs:
...
-        spec: cypress/e2e/${{ env.TEST_SUITE }}/*.js
+        command: npx cypress run --spec cypress/e2e/${{ env.TEST_SUITE }}/*.js --record --parallel --key ${{ secrets.CYPRESS_MANUAL_RECORD_KEY }}  
         group: 'e2e Tests'
       env:
         TEST_SUITE: ${{ github.event.inputs.suite }}
         # For recording and parallelization to work you must set your CYPRESS_RECORD_KEY
         # in GitHub repo → Settings → Secrets → Actions
-        CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+        CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_MANUAL_RECORD_KEY }}
+        CYPRESS_PROJECT_ID: ${{ secrets.CYPRESS_MANUAL_PROJECT_ID }}
```
Comment (bilalmajid-deriv) [inline .github/workflows/cypress_cloud_manual.yml:54]: can we now also remove the projectId from cypress.config.js ?

Comment (meeyun-deriv) [reply .github/workflows/cypress_cloud_manual.yml:54]: the smoketest.yml from PR trigger is using the default projectId, so we still need it in `cypress.config.js`

File: .github/workflows/cypress_cloud_manual.yml [hunk 2]
```diff
@@ -44,13 +44,15 @@ jobs:
         # https://docs.cypress.io/guides/cloud/projects#Set-up-a-project-to-record
         record: true
         parallel: true # Runs test in parallel using settings above
-        spec: cypress/e2e/${{ env.TEST_SUITE }}/*.js
+        command: npx cypress run --spec cypress/e2e/${{ env.TEST_SUITE }}/*.js --record --parallel --key ${{ secrets.CYPRESS_MANUAL_RECORD_KEY }}
```
Comment (maliheh-deriv) [inline .github/workflows/cypress_cloud_manual.yml:47]: @meeyun-deriv shouldn't we declare the variables for security concerns ?

Comment (meeyun-deriv) [reply .github/workflows/cypress_cloud_manual.yml:47]: yes we should, we can do it in the next PR to update all workflow which has the same issue

**Review by maliheh-deriv — APPROVED** [review]

**Pull request merged**